### DISCUSSION TITLE
Integration of Nextcloud Talk recording service

### DISCUSSION THREAD

**Discussion by nr-plaxon** [nextcloud/all-in-one#2527] [Ideas] [answered]
It would be great if the Nextcloud Talk **recording** service (see [feature announcement for HUB4](https://www.youtube.com/watch?v=JGJ5Bqoe2WA)) as part of the spreed signaling server cloud be implemented in Nextloud AIO.
https://github.com/nextcloud/spreed/pull/9177

**Comment by szaimen** [comment]
I guess we can look into this after Nc26 is realeased for everyone. See https://github.com/nextcloud/all-in-one/issues/2336

**Comment by szaimen** [comment]
Possibly needs 
- https://github.com/nextcloud/spreed/pull/9174
- https://github.com/nextcloud/spreed/pull/9177

**Reply by Zoey2936** [reply]
>      [Docker compose for recording server spreed#9177](https://github.com/nextcloud/spreed/pull/9177)

I think that could also be done on alpine.

**Reply by szaimen** [reply]
> I think that could also be done on alpine.

I think so as well 👍

**Reply by szaimen** [reply]
btw @Zoey2936 would you be up for creating the Dockerfile for recording server? I would then tackle changes that are needed in the mastercontainer.

**Reply by Zoey2936** [reply]
can you say me please, where I find documentation for this?

**Reply by nr-plaxon** [reply]
This might help you get started:
https://github.com/nextcloud/spreed/blob/8d75b7cab773d519dc66c8034086b47189aef5de/recording/docker-compose/Dockerfile

Thanks for tackling this! It would be so great to have this feature available in AIO.

**Reply by szaimen** [reply]
Indeed. Unfortunately we dont have much more docs available than the files in https://github.com/nextcloud/spreed/tree/8d75b7cab773d519dc66c8034086b47189aef5de/recording

**Comment by szaimen** [comment, marked as answer]
The feature will be available in the next AIO release.

**Reply by szaimen** [reply]
This is now released with v6.1.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
